src/App.jsx

```diff
@@ -1,16 +1,45 @@
-import React from 'react';
-import { registerRootComponent } from 'expo';
+import React, { useState, useEffect } from 'react';
+import { registerRootComponent, AppLoading } from 'expo';
+import * as Font from 'expo-font';
 import { Provider } from 'react-redux';
 import { createAppContainer } from 'react-navigation';
+import { Toast } from 'native-base';
 import AppNavigator from './Navigator';
-import store from './redux/store';
+import store from './reduxStore';
+import Roboto from '../assets/Fonts/Roboto/Roboto-Regular.ttf';
+import RobotoMedium from '../assets/Fonts/Roboto/Roboto-Medium.ttf';
 
 const AppContainer = createAppContainer(AppNavigator);
 
-const App = () => (
-  <Provider store={store}>
-    <AppContainer />
-  </Provider>
-);
+const App = () => {
+  const [fontLoading, setFontLoading] = useState(true);
+
+  useEffect(() => {
+    async function loadFont() {
+      try {
+        await Font.loadAsync({
+          Roboto,
+          Roboto_medium: RobotoMedium,
+        });
+        setFontLoading(false);
+      } catch (err) {
+        Toast.show({
```

Comment:
this will fail again since the Toast will need the font what we couldnt load :)

Reply:
fixed with Alert